app/Http/Controllers/Abilities/AbilityReorderController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Controllers\Abilities;
```

Comment:
I think it would make more sense to put this in the Entity namespace, since it's related to an entity's abilities :)